packages/flutter/lib/src/widgets/text.dart

```diff
@@ -201,6 +201,77 @@ class DefaultTextStyle extends InheritedTheme {
   }
 }
 
+/// The [TextHeightBehavior] that will apply to descendant [Text] widgets which
+/// have not explicitly set [Text.textHeightBehavior].
+///
+/// See also:
+///
+///  * [DefaultTextStyle], which defines a [TextStyle] to apply to descendant
+///    [Text] widgets.
+class DefaultTextHeightBehavior extends InheritedTheme {
+  /// Creates a default text height behavior for the given subtree.
+  ///
+  /// This will take precedence over [DefaultTextStyle.textHeightBehavior].
```

Comment:
We usually don't put vital info like this in the constructor's API doc

Reply:
Moved